onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -234,6 +235,11 @@ struct MyCustomOp : Ort::CustomOpBase<MyCustomOp, MyCustomKernel> {
   void* CreateKernel(Ort::CustomOpApi api, const OrtKernelInfo* info) { return new MyCustomKernel(api, info); };
   const char* GetName() const { return "Foo"; };
 
+#ifdef USE_CUDA
+  // Make the kernel run on CUDA for CUDA builds
```

Comment:
The custom op kernel has a Compute() method that doesn't really use CUDA, but that is not really required for the purpose of this exercise. We only need to verify if this node is assigned to the CUDA EP (which previously wouldn't have been the case.)

Reply:
Would be helpful if at least some of this info was in a comment in the file. 

---
In reply to: [389223648](https://github.com/microsoft/onnxruntime/pull/3165#discussion_r389223648) [](ancestors = 389223648)

Reply:
Needed to slightly adjust the test to account for the failing builds. Adjusted the test and added a detailed comment on what exactly we are testing for.